stable/nginx-ingress/templates/clusterrole.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create -}}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
As I understand it, the charts repo supports Kubernetes version N and N-1. Right now that versions 1.8 and 1.7. So, the charts need to work with both versions. The 1.7 API supports v1beta1 and not v1. So, this should be merged when 1.9 comes out.

cc: @kubernetes/charts-maintainers

Reply:
+1 we should keep this as v1beta1 for now

Reply:
actually chart supports ONLY N-1(1.7) k8s version
what to do with k8s version N(1.8)?

Reply:
@stamak [According to the deprecation policy, the beta API will be around for 2 releases or 6 months](https://kubernetes.io/docs/reference/deprecation-policy/). So, the `v1beta1` API will still work for 1.8.

Reply:
thx